src/components/communities/CommunityAdministrationPage.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useEffect, useState } from "react";
-import { ScrollView, Switch, StyleSheet, Alert, Button } from "react-native";
+import { ScrollView, Switch, StyleSheet, Alert } from "react-native";
 import { SubmitHandler, useForm } from "react-hook-form";
+import { useTheme } from "@shopify/restyle";

```

Comment:
do we need this line? I think you import `theme` below and I haven't seen us use this hook in the codebase yet

Reply:
This is used in all the themed components. The import of Theme below is the type of theme we define in theme.tsx. I am using it here to give our 'red' buttons consistency.

Reply:
ah I see. I've been just importing the `theme` object directly from theme.tsx but this is probably better